nixos/modules/services/web-apps/nextcloud.nix

```
@@ -868,7 +868,8 @@ in {
         ++ (optional (versionOlder cfg.package.version "27") (upgradeWarning 26 "23.11"))
         ++ (optional (versionOlder cfg.package.version "28") (upgradeWarning 27 "24.05"))
         ++ (optional (versionOlder cfg.package.version "29") (upgradeWarning 28 "24.11"))
-        ++ (optional (versionOlder cfg.package.version "30") (upgradeWarning 29 "24.11"));
+        ++ (optional (versionOlder cfg.package.version "30") (upgradeWarning 29 "24.11"))
+        ++ (optional (versionOlder cfg.package.version "31") (upgradeWarning 30 "25.05"));
```

```suggestion
        ++ (optional (versionOlder cfg.package.version "31") (upgradeWarning 30 "25.05"))
        ;
```
would make future diffs prettier